scripts/summarise_all_rels_im.py

```diff
@@ -25,35 +29,36 @@
         "--median",
         dest="stat_fn",
         action="store_const",
-        const=np.median,
-        default=np.mean,
+        const="median",
+        default="mean",
         help="find median (default: mean)",
     )
     parser.add_argument(
         "--im",
         dest="im_types",
         nargs="+",
         type=str,
+        choices=ALL_IMS,
```

Comment:
When using this, I'm having some issues with pSA, as it gets a KeyError with pSA in the columns. Maybe should add each im for pSA with the period to this?

Reply:
You want to explicitly list all 132 pSA periods?
Can I see the help output for that one? :P